### DISCUSSION TITLE
What are "Suggested translations" on the dashboard?

### DISCUSSION THREAD

**Discussion by ib** [WeblateOrg/weblate#12445] [Q&A] [answered]
I don't quite understand what language components are being presented to me, on what basis, and why?

**Comment by nijel** [comment, marked as answer]
These are translations in the language you translate to that are not yet completed.

**Comment by ib** [comment]
And is there any selection or limitation? There must be many more incomplete projects than the 10 suggested to me.

**Reply by nijel** [reply]
Yes, it shows random 10 of them.

**Comment by lucasmz-dev** [comment]
I've been annoyed that this tab is like stuck in time ever since I created my Weblate account. None of those projects end up interesting me enough for me to be like 'I should translate this', I don't like to translate things I don't understand

**Reply by nijel** [reply]
The selection is really naive now and does not look at anything besides the project activity and language you translate. It works for small Weblate instances, but you will rarely get interesting selection on Hosted Weblate.
